### PR TITLE
Chore/update api paths

### PR DESCRIPTION
**Pull request by ryan-haley-code42**
### Description of Change ###

updates core paths to use api/v# 

### Testing Procedure ###
endpoints hit directly with v1 pathing via sdk library

### PR Checklist ###
Did you remember to do the below?

- [ x] Add an entry to CHANGELOG.md describing this change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Removed method Get_alert_log(), method was no longer functional
 
+### Changed
+
+- updated uri paths to use updated `/api/v#`
```
Comment (timabrmsn) [inline CHANGELOG.md:19]: To give customers who might see this a clearer picture of why this change, I'd say: `updated URI paths to use new standardized versioning scheme`

**Review by timabrmsn — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ryan-haley-code42) [reply CHANGELOG.md]: agreed, updated

**Comment by ryan-haley-code42** [comment]
integration tests pass with local mock server main branch
